harness/determined/pytorch/_pytorch_context.py

```diff
@@ -129,6 +129,12 @@ def get_lr_scheduler(self) -> Optional[pytorch.LRScheduler]:
             return self.lr_schedulers[0]
         return None
 
+    def get_use_amp(self):
+        return self._use_amp
+
+    def get_amp_state(self):
+        return apex.amp.state_dict()
```

Comment:
Probably the caller should just call `apex.amp.state_dict()`, since that is a global function anyway... it doesn't have anything to do with the PyTorchTrialContex object at all.